build.gradle

```diff
@@ -147,8 +147,7 @@ distributions {
 
 
 install4j {
-    if (install4jHomeDir == null) installDir = file("foo")
-    else installDir = file(install4jHomeDir)
+    if (hasProperty('install4jHomeDir'))  installDir = file(install4jHomeDir)
```

Comment:
I have to revert this change as it breaks my gradle task.
It seems install4j refuse to run if during the configuration phase variable installDir is not set this is why it has to be also specified in else clause

Reply:
> 
> 
> I have to revert this change as it breaks my gradle task.
> It seems install4j refuse to run if during the configuration phase variable installDir is not set this is why it has to be also specified in else clause

hi @akasolace : could you try it this way: 

    if (hasProperty('install4jHomeDir'))  installDir = file(install4jHomeDir)
    else installDir = file("foo")

this would work in my case.

Reply:
@wsbrenk arf not yet but the following should (hopefully) work for both of us :-)
```
    if ((hasProperty('install4jHomeDir')) || ((install4jHomeDir != null) )) installDir = file(install4jHomeDir)
    else installDir = file("foo")
```

Reply:
nope: A problem occurred evaluating root project 'HO'.
> path may not be null or empty string. path=''

what's about this on your side?

    if (hasProperty('install4jHomeDir')){
        if (install4jHomeDir != null) installDir = file(install4jHomeDir)
        else installDir = file("foo")
    }